mlflow/sagemaker/cli.py

```diff
@@ -24,7 +24,7 @@ def commands():
 @click.option("--bucket", "-b", default=None, help="S3 bucket to store model artifacts")
 @cli_args.RUN_ID
 @click.option("--image-url", "-i", default=None, help="ECR URL for the Docker image")
-@click.option("--region-name", "-r", default="us-west-2",
+@click.option("--region-name", "-rn", default="us-west-2",
```

Comment:
multi-letter shortenings are odd (sometimes `-rn` would be equivalent to `-r -n`), so maybe we can use something else, or remove the shortening.

Reply:
Good point. Removed instances of multi-letter shortening.